packages/icon-button/example/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Jøkul Button Example</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="index.scss" />
+    </head>
+
+    <body>
+        <button class="jkl-button jkl-button--primary">
+            En primærknapp med masse tekst
+        </button>
+        <button class="jkl-button jkl-button--secondary">
+            Kort
+        </button>
+        <button class="jkl-button jkl-button--tertiary">
+            Tertiærknapp
+        </button>
+    </body>
+</html>
```

Comment:
This is the example from `jkl-button`. We don't really use the examples in the style packages anymore, so you can just delete it if you want (we should probably clean this up in the other packages as well).

Reply:
removed: 7f87ae7dc5c94dd7be828f27288f8db7ed29b314